documentation/plain_rules.md

```diff
@@ -183,7 +183,7 @@ Note that you can encode a JSON in the `template` field:
     }
 ```
 
-The `template` field performs [string substitution](#string-substitution-syntax).
+The `template` and `url` fields perform [string substitution](#string-substitution-syntax).
```

Comment:
I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.